### PR TITLE
GOLD-277 : Required minimum for consensus should be STATIC not dynamic number

### PR DESCRIPTION
**Pull request by abdulazeem-tk4vr**
Linear: https://linear.app/shm/issue/GOLD-277/required-minimum-for-consensus-should-be-static-not-dynamic-number

### CODE REVIEW

File: src/snapshot/index.ts [hunk 1]
```diff
@@ -399,10 +399,15 @@ export async function safetySync() {
   })
 
   // Figure out which nodes hold which partitions in the new network
+  const minNodesPerConsensuGroup = 32
```
Comment (timadinorth) [inline src/snapshot/index.ts]: magic number ? why not have parameter in config with static number ?

Comment (abdulazeem-tk4vr) [reply src/snapshot/index.ts]: I am a bit confused how this is any different than this in the config. 
```
  sharding: { nodesPerConsensusGroup: 5, nodesPerEdge: 2, executeInOneShard: false },
  ```

Comment (abdulazeem-tk4vr) [reply src/snapshot/index.ts]: image

Comment (abdulazeem-tk4vr) [reply src/snapshot/index.ts]: What I meant to say was why cant we set `config.sharding.nodesPerConsensusGroup` to a static value like 32 ?
Why is there a need to create a new variable? 
Do we want 2 config variables? 
- One is `config.sharding.nodesPerConsensusGroup` which is dynamic and can change
- the other is `config.sharding.minNodesPerConsensusGroup` which is constant and set to 32. 

We shall then use the max of these 2 values. Does this sound good?

Comment (abdulazeem-tk4vr) [reply src/snapshot/index.ts]: I am going to make this change.

**Pull request closed without merging**